modules/nw-metallb-configure-address-pool.adoc

```diff
@@ -12,27 +12,29 @@ As a cluster administrator, you can add address pools to your cluster to control
 
 .Procedure
 
-. Create a file, such as `addresspool.yaml`, with content like the following example:
+. Create a file, such as `ipaddresspool.yaml`, with content like the following example:
 +
 [source,yaml]
 ----
-apiVersion: metallb.io/v1alpha1
-kind: AddressPool
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
 metadata:
-  namespace: metallb-system
+  namespace: openshift-operators
```

Comment:
Leaving the comment here but it's true for all the other CRDs: we need to declare that this must be the ns metallb was deployed to, and I'd stick with the old value (basically reverting all these changes)